manager/catalog_manager.go

```diff
@@ -26,10 +36,11 @@ func (i *arrayFlags) Set(value string) error {
 }
 
 var (
-	refreshInterval = flag.Int64("refreshInterval", 60, "Time interval (in Seconds) to periodically pull the catalog from git repo")
-	logFile         = flag.String("logFile", "", "Log file")
-	debug           = flag.Bool("debug", false, "Debug")
-	validate        = flag.Bool("validate", false, "Validate catalog yaml and exit")
+	refreshInterval  = flag.Int64("refreshInterval", 60, "Time interval (in Seconds) to periodically pull the catalog from git repo")
+	logFile          = flag.String("logFile", "", "Log file")
+	debug            = flag.Bool("debug", false, "Debug")
+	validate         = flag.Bool("validate", false, "Validate catalog yaml and exit")
+	catalogURLBranch = flag.String("catalogURLBranch", "", "Branch name")
```

Comment:
The value of catalogURLBranch changes for every URL in the for loop and is entered into the catalogCollection. I declared it here so as to declare all variables together.